utils/generate_grammar_tables.py

```diff
@@ -134,32 +134,71 @@ def __str__(self):
             ref_type_id=(1 if self.ref_type_id else 0))
 
 
-def generate_instruction(inst):
+class ExtInstInitializer(object):
+    """Prints a SPIR-V instruction as the initializer for
```

Comment:
Nitpick: 
Prints is not right here, really.  This is a class definition. Instances convert to a string suitable for printing.